src/status_im/contexts/centralized_metrics/tracking.cljs

```diff
@@ -62,4 +78,15 @@
     :set-view-id
     (track-view-id-event second-parameter)
 
+    :wallet/select-account-tab
+    (when (= second-parameter :collectibles)
+      (navigated-to-collectibles-tab-event :account))
```

Comment:
`case` here checks the `event-name` parameter,  while `second-parameter` can be any because it is the second parameter of re-frame event that we are intercepting (agree that new name needed). For different case clauses `second-parameter` can be different or absent. So I'm not sure `cond` will work here because condition will look like `(and (= event-name :wallet/select-account-tab) (= second-parameter :collectibles)`. Wdyt?